Rakefile

```diff
@@ -13,7 +13,11 @@ Rake::TestTask.new do |test|
   test.verbose = true
 end
 
-RuboCop::RakeTask.new
+RuboCop::RakeTask.new do |t|
+  # allow you to run "$ rake rubocop -a" to autofix
```

Comment:
prefer something like
```
RuboCop::RakeTask.new :fmt do |t|
  t.options << '-A'
end
```
or a `rubocop:reformat` etc, but might refactor that later

Reply:
I wouldn't default to -A and it is unsafe. 

RuboCop::RakeTask.new created three tasks. 
```
$ rake -T
rake rubocop                  # Run RuboCop
rake rubocop:autocorrect      # Autocorrect RuboCop offenses (only when it's safe)
rake rubocop:autocorrect_all  # Autocorrect RuboCop offenses (safe and unsafe)
```

So I could use `rake rubocop:autocorrect`  to format my code but. The way I work is that I make some changed to the code without thinking to much about style, then run specs and I ran the specs with `rake` (the default task) and in those cases it would stop at rubocop. In anouther project I added these two options so I could run `rake -a`. I could also change to run `rake  test`. 

So maybe I should just drop this change